kafka/metrics.go

```diff
@@ -407,23 +419,31 @@ func (h *metricHooks) OnBrokerWrite(meta kgo.BrokerMetadata, key int16, bytesWri
 	)
 }
 
-func (h *metricHooks) OnBrokerRead(meta kgo.BrokerMetadata, _ int16, bytesRead int, _, _ time.Duration, err error) {
+func (h *metricHooks) OnBrokerRead(meta kgo.BrokerMetadata, _ int16, bytesRead int, readWait, timeToRead time.Duration, err error) {
 	attrs := make([]attribute.KeyValue, 0, 2)
```

Comment:
To avoid an extra allocation, I think we should bump the slice capacity from 2 to 3
```suggestion
	attrs := make([]attribute.KeyValue, 0, 3)
```

Reply:
updated